Makefile

```diff
@@ -0,0 +1,69 @@
+.PHONY: all clean tarball srpm rpm
+
+PACKAGE_NAME = perl-WWW-Gooddata
+
+VERSION ?= $(shell git describe --tags --match '*.*.*' | tr -- '-' '_')
```

Comment:
Looks like version is also at lib/WWW/GoodData.pm not sure if we don't need to update it also or just don't do this automation from git tag. This looks like repo that will see very litle commits once its cleaned up to proper state. So manual release by updating lib/WWW/GoodData.pm and taking version from there might be solution